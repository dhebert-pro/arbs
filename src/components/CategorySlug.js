import React from "react";

const CategorySlug = ({ slug }) => <div>La cat√©gorie est : {slug}</div>;

export default CategorySlug;
