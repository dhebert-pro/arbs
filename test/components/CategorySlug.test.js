/*global expect test describe:true */
/*eslint no-undef: "error"*/
import React from "react";
import { shallow } from "enzyme";
import CategorySlug from "components/CategorySlug";
import renderer from "react-test-renderer";

describe("CategorySlug", () => {
  test("should be rendered", done => {
    const component = renderer.create(<CategorySlug slug="TEST" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    done();
  });
  test("should render slug", done => {
    const component = shallow(<CategorySlug slug="TEST" />);
    expect(component.find("div").length).toEqual(1);
    expect(component.find("div").text()).toEqual("La cat√©gorie est : TEST");
    done();
  });
});
