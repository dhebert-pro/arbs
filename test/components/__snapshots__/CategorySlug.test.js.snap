// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategorySlug should be rendered 1`] = `
<div>
  La cat√©gorie est : 
  TEST
</div>
`;
